Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader

```diff
@@ -0,0 +1,177 @@
+﻿//
```

Comment:
Should we be adding additional shaders to the toolkit?

Could we just make a new material out of existing shaders?